CODE_OF_CONDUCT.md

```diff
@@ -65,6 +65,10 @@ Project maintainers who do not follow or enforce the Code of Conduct in good
 faith may face temporary or permanent repercussions as determined by other
 members of the project's leadership.
 
+To learn more about the process of how moderators enforce the Code of Conduct, 
+review the [Moderation Policy](https://github.com/nodejs/TSC/blob/master/Moderation-Policy.md#moderation-policy)
+and [Requesting Moderation](https://github.com/nodejs/TSC/blob/master/Moderation-Policy.md#requesting-moderation).
```

Comment:
Can you please move the actual urls to the bottom index and use the `[Moderation Policy][]` syntax inline? :-)